.github/workflows/ci.yml

```diff
@@ -156,6 +157,14 @@ jobs:
           target: x86_64-apple-darwin
           rust: stable
           os: macos-latest
+        - thing: x86_64-mingw
+          target: x86_64-pc-windows-gnu
+          rust: stable
+          os: windows-latest
+          custom_env:
+            CC: gcc
+            CXX: g++
+            PATH: "C:\msys64\usr\bin;$env:PATH"
```

Comment:
I think you need to escape this if you put it in quotes. (I also doubt the `$env:PATH` part will work, the env setting happens before GitHub launches the shell.)

Reply:
Yep and yep. One more attempt, for science.